File: covid-api/src/main/java/com/tacs/grupo1/covid/api/dto/UserInformation.java
package com.tacs.grupo1.covid.api.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.Accessors;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Accessors(chain = true)
public class UserInformation {
    @JsonProperty("id")
    private Long id;
    @JsonProperty("user_name")
    private String userName;
    @JsonProperty("telegram_id")
    private Long telegramId;
    @JsonProperty("name")
    private String name;
    @JsonProperty("last_name")
    private String lastName;
    @JsonProperty("email")
    private String email;
    @JsonProperty("total_lists")
    private long totalLists;
    @JsonProperty("total_countries")
    private long totalCountries;
    @JsonProperty("last_login")
    private LocalDateTime lastLogin;
}

File: covid-frontend/src/models/UserInformation.ts
export default class UserInformation{
    public id: number;
    public user_name: string;
    public telegram_id: string;
    public name: string;
    public last_name: string;
    public email: string;
    public total_lists: number;
    public total_countries: number;
    public last_login: string;

    constructor(userInformation?: any){
        if(userInformation){
            this.id = userInformation.id;
            this.user_name = userInformation.user_name;
            this.name = userInformation.name;
            this.last_name = userInformation.last_name;
            this.email = userInformation.email;
            this.last_login = userInformation.last_login;
            this.telegram_id = userInformation.telegram_id;
            this.total_lists = userInformation.total_lists;
            this.total_countries = userInformation.total_countries;
        }else{
            this.id = 0;
            this.user_name = "";
            this.name = "";
            this.last_name = "";
            this.email = "";
            this.last_login = "";
            this.telegram_id = "";
            this.total_lists = 0;
            this.total_countries = 0;
        }
    }
}